<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>restim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="PhaseWidget" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Pattern generator</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Mouse</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Circle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Audio</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCommandLinkButton" name="startStopAudioButton">
            <property name="text">
             <string>Start audio</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="TransformCalibrationSettingsWidget" name="tab_transform_calibration">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
      </widget>
      <widget class="ModulationSettingsWidget" name="tab_carrier">
       <attribute name="title">
        <string>Carrier and modulation</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_volume">
       <attribute name="title">
        <string>Volume</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="VolumeControlWidget" name="volumeWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="WaveformDetailsWidget" name="tab_details">
       <attribute name="title">
        <string>Details</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFunscript_conversion_2"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionFunscript_conversion_2">
   <property name="text">
    <string>Funscript conversion</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModulationSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/modulationsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhaseWidget</class>
   <extends>QGraphicsView</extends>
   <header>qt_ui/phasewidget.h</header>
  </customwidget>
  <customwidget>
   <class>WaveformDetailsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/waveformdetailswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransformCalibrationSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/transformcalibrationsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeControlWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/volumecontrolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
